bootstrap/kubeadm/api/v1beta1/kubeadm_types.go

```diff
@@ -222,6 +225,11 @@ type BottlerocketKernelSettings struct {
 	SysctlSettings map[string]string `json:"sysctlSettings,omitempty"`
 }
 
+// BottlerocketBootSEttings holds the boot-related settings for bottlerocket nodes.
+type BottlerocketBootSettings struct {
+	KernelParameters map[string][]string `json:"kernelParameters,omitempty"`
```

Comment:
Can we rename this to `BootKernelParameters` since having another kernel parameters name might be confusing between the one under `BottlerocketSettings`